internal/csi-common/server.go

```diff
@@ -97,7 +97,7 @@ func (s *nonBlockingGRPCServer) serve(endpoint string, srv Servers) {
 		klog.Fatalf("Failed to listen: %v", err)
 	}
 
-	server := grpc.NewServer()
+	server := grpc.NewServer(NewMiddlewareServerOption(false))
```

Comment:
@nixpanic can we make a new PR from this commit and get it merged? not sure somehow we missed how it got removed

Reply:
sure!

Reply:
See #4263. Will drop it from this PR once the e2e run finishes.